test/integration/developers_test.rb

```diff
@@ -47,6 +47,7 @@ class DevelopersTest < ActionDispatch::IntegrationTest
     end
 
     assert user.developer.role_type.part_time_contract?
+    assert_equal "Lovelace.jpg", user.developer.avatar.filename.to_s
```

Comment:
Let's rename this to `lovelace.jpg` (without the capital "L") to match the rest of the files.